src/screens/LocationPage/LocationRouter.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { useParams, Redirect } from 'react-router-dom';
+import { getCountyByUrlName, getStateByUrlName } from 'common/locations';
+import LocationPage from './LocationPage';
+
+interface LocationPageUrlParams {
+  stateId: string;
+  countyId: string | undefined;
+}
+
+const LocationRouter: React.FC = () => {
```

Comment:
It seems like maybe this isn't actually used?  Or am I confused?

Reply:
It is, it's the default export in `LocationPage/index.ts` - before this PR, the AppBar was redirecting to the homepage when the state or county didn't exist (it took me a while to discover that), I moved that logic here to make it easier to find... maybe I should move this to `LocationPage.js` instead?

Reply:
oops! Yeah, I was confused.  This is probably fine, but I'd be fine moving it into LocationPage.js too. Up to you.